test/fixtures/line/shadow.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
               yMax: 8,
               borderColor: 'blue',
               borderWidth: 4,
-              backgroundShadowColor: 'black',
+              borderShadowColor: 'black',
```

Comment:
Why are these needed and does the fixture actually catch an missing shadow in this test?

Reply:
backgroundColor is not needed because as suggested, has been moved to arrow heads config.
borderColor is needed to apply the shadow to the line.
And yes, the fixture catches any missing shadow.